runtime/v2/bundle.go

```diff
@@ -115,16 +116,24 @@ type Bundle struct {
 }
 
 // Delete a bundle atomically
-func (b *Bundle) Delete() error {
+func (b *Bundle) Delete(ctx context.Context) error {
```

Comment:
What's the likelihood we miss a shim panic log by only capturing them on bundle deletion?

Reply:
As far as my understanding goes this should practically cover all of the cases. The only case that I can think of is if containerd doesn't cleanup the bundle directory even after the shim has panicked but I think that means we have some other bugs in containerd if that's happening.